versioned_docs/version-0.9/user-guide/deployments/configuration.md

```diff
@@ -319,6 +320,7 @@ backoff_deadline = "5mins"
 - `backoff_max`: The maximum backoff delay.
 - `backoff_base`: The exponential backoff rate.
 - `backoff_deadline`: The deadline of retries.
+- `overwrite_entry_start_id`: This option ensures that when Kafka messages are deleted, the system can still successfully replay memtable data without throwing an out-of-range error. However, enabling this option might lead to unexpected data loss, as the system will skip over missing entries instead of treating them as critical errors.
```

Comment:
```suggestion
- `overwrite_entry_start_id`: This option ensures that when Kafka messages are accidentally deleted, the system can still successfully replay memtable data without throwing an out-of-range error. However, enabling this option might lead to unexpected data loss, as the system will skip over missing entries instead of treating them as critical errors.
```